src/main/webapp/pages/projects/settings/pages/remote.html

```diff
@@ -1,69 +1,21 @@
 <!DOCTYPE html>
-<html xmlns:th="http://www.thymeleaf.org" xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
-      data-layout-decorate="~{projects/settings/_base}">
-<head>
-    <title th:text="#{project.settings.page.title.remote}">Title</title>
-        <script th:inline="javascript">
-        var PAGE = {
-            urls: {
-            'sync': /*[[@{/projects/{id}/settings/sync(id=${project.getId()})}]]*/ '/projects/1/settings/sync',
-            }
-        };
-        </script>
-</head>
-<body>
-<div layout:fragment="settings-content" th:with="status = ${project.getRemoteStatus()}">
-    <h2 class="h2-muted" th:text="#{project.settings.sync.heading}"></h2>
-    <div class="form-group">
-        <label class="control-label" th:text="#{project.settings.sync.lastSync}">_Last Sync_</label>
-        <div>
-            <span th:text="${#calendars.format(project.getRemoteStatus().getLastUpdate(), 'dd MMM yyyy hh:mma')}"></span>
-            <button id="forceSync" role="button" class="btn btn-default btn-sm"
-                    th:text="#{project.settings.sync.forceSync}">_Force Sync_
-            </button>
-        </div>
+<html
+  lang="en"
+  xmlns:th="http://www.thymeleaf.org"
+  xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+  data-layout-decorate="~{projects/settings/_base}"
+>
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="/dist/css/project-remote.bundle.css">
+    <title th:text="#{ProjectRemoteSettings.title}">Title</title>
+  </head>
+  <body>
+    <div id="project-remote-root" layout:fragment="settings-content" >
+      <div th:text="#{ProjectRemote.loading}">LOADING ...</div>
```

Comment:
We should create a thymeleaf fragement that displays an internationalizaed loading component, before react takes over.  You don't need to do this here.

Reply:
Good idea!